packages/playwright/src/reporters/teleEmitter.ts

```diff
@@ -294,10 +295,18 @@ export class TeleReporterEmitter implements ReporterV2 {
       duration: step.duration,
       error: step.error,
       attachments: step.attachments.length ? step.attachments.map(a => result.attachments.indexOf(a)) : undefined,
-      annotations: step.annotations.length ? step.annotations : undefined,
+      annotations: step.annotations.length ? this._relativeAnnotationLocations(step.annotations) : undefined,
     };
   }
 
+  private _relativeAnnotationLocations(annotations: TestAnnotation[]): TestAnnotation[] {
+    return annotations.map(annotation => {
+      if (annotation.location)
+        annotation.location = this._relativeLocation(annotation.location);
```

Comment:
I think updating the `annotation` object inline is not what we want here. It comes from the public API and may be used by other reporters.